metals/src/main/scala/scala/meta/internal/metals/BuildTargets.scala

```diff
@@ -226,6 +237,28 @@ final class BuildTargets() {
   ): Option[ScalacOptionsItem] =
     scalacTargetInfo.get(buildTarget)
 
+  /**
+   * Returns meta build target for `*.sbt`  files.
+   * It selects build target by directory of its connection
+   *   because `*.sbt` aren't included in `sourceFiles` set
+   */
+  def sbtBuildScalaTarget(file: AbsolutePath): Option[ScalaTarget] = {
```

Comment:
Do we return the right sbt target for the Scala files located in `project` directories? I think if Bloop doesn't provide use with proper source roots we could add a custom logic for resolving target by source file in `inverseSources`. If no build target is found we can look if this file should belong to an sbt build target. What do you think?

Reply:
This method is only for `.*sbt` files.
For `*.scala` files from meta build `inverseSources` is used as usual.

I think when we start implementing compilation for `.*sbt` files it would be better to move this logic into `inverseSources` method. But for now, I'm not sure if there is a need for that.